app/internal/user.py

```diff
@@ -0,0 +1,46 @@
+from app.database import models, schemas
+from app.internal.security.ouath2 import get_hashed_password
+from sqlalchemy.orm import Session
+
+
+def get_by_id(db: Session, user_id: int) -> models.User:
+    """query database for a user by unique id"""
+    return db.query(models.User).filter(models.User.id == user_id).first()
+
+
+def get_by_username(db: Session, username: str) -> models.User:
```

Comment:
How does this function is different from `get_db_user_by_username`?

Reply:
I can use this one, but i tried to explain above why i did that.